rosdep/python.yaml

```diff
@@ -3957,16 +3957,7 @@ python-rosinstall-generator:
   rhel:
     '7': [python2-rosinstall_generator]
   ubuntu:
-    lucid: [python-rosinstall-generator]
-    maverick: [python-rosinstall-generator]
-    natty: [python-rosinstall-generator]
-    oneiric: [python-rosinstall-generator]
-    precise: [python-rosinstall-generator]
-    quantal: [python-rosinstall-generator]
-    raring: [python-rosinstall-generator]
-    saucy: [python-rosinstall-generator]
-    trusty: [python-rosinstall-generator]
-    trusty_python3: [python3-rosinstall-generator]
```

Comment:
I think @tfoote wanted to leave these `_python3` keys to avoid breaking people using them. Then again this is a trusty key so maybe we can retire it.

Reply:
I see. I'm fine in either way, will wait for direction.

Indeed EoL doesn't mean there are no users.

Reply:
Let's just put the `trusty_python3` key back as it was.  All of the others will be subsumed by the wildcard matching.  While I would like to get rid of the old keys, this is the safest thing to do.

Reply:
I think we can remove the EOL `trusty_python3` key: https://github.com/ros/rosdistro/pull/25120#discussion_r432116597

Reply:
Thanks...So, this PR is now finalized?

> Indeed EoL doesn't mean there are no users.

I might have been confused. I guess my statement above is always true but that's what EoL deals with.

Reply:
> So, this PR is now finalized?

From an author's perspective it is finalized for now but I guess we're waiting for the discussion in https://github.com/ros/rosdistro/pull/25507 to finalize?